src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -758,8 +781,17 @@ BookStep<TIn, TOut, TDerived>::forEachOffer(
     };
 
     // At any payment engine iteration, AMM offer can only be consumed once.
-    auto tryAMM = [&](std::optional<Quality> const& quality) -> bool {
-        auto ammOffer = getAMMOffer(sb, quality);
+    auto tryAMM = [&](std::optional<Quality> const& lobQuality) -> bool {
+        // If offer crossing then use the best quality out of LOB and
+        // limit quality to prevent AMM being blocked by a lower quality
+        // LOB.
+        auto const bestQuality = [&]() -> std::optional<Quality> {
+            if (sb.rules().enabled(fixAMMRounding) && lobQuality)
```

Comment:
This change doesn't relate to "AMMRounding". We might want to rename the feature.

Reply:
For NFTs there was a rollup of three different fixes named `fixNonFungibleTokensV1_2`.  Perhaps something like `fixAMMv1_1` would work?

Reply:
We could think about rolling up the other AMM rounding amendment that we added to 2.2 into this one as well.

Reply:
We only have `fixAMMRounding` in 2.2, no?

Reply:
You're right. My bad.